src/pages/border-information/index.js

```diff
@@ -31,22 +27,14 @@ const BorderInformationPage = () => {
           title={t("How to cross the border")}
           subcomponent={
             <section className="mt-10 text-center">
-              <p className="text-xl font-semibold">
-                {t("Choose a country")}:
-              </p>
+              <p className="text-xl font-semibold">{t("Choose a country")}:</p>
               <CountryPicker {...{ availableCountries, selectedCountry }} />
             </section>
           }
         />
       }
     >
-      {data && <BorderCrossingInfo
-        title={t(
-          "Information for Ukrainian citizens travelling {{to_country}}",
-          { to_country: toName },
-        )}
-        data={data}
-      />}
+      {data && <BorderCrossingInfo title={t("Information for Ukrainian citizens travelling {{to_country}}", { to_country: toName })} data={data} />}
```

Comment:
There is a missing closing circle bracket ")"

Reply:
t("Information for Ukrainian citizens travelling {{to_country}}")

Reply:
Hmm I see it's closing after the to_country param
`t("Information for Ukrainian citizens travelling {{to_country}}", { to_country: toName })`